### DISCUSSION TITLE
24h shares statistics for miner

### DISCUSSION THREAD

**Discussion by coolman888** [coinfoundry/miningcore#1038] [Q&A]
Does anyone know how to do 24h shares statistics for miner with Valid/Stale/Invalid data? I can not find these data from API.
Thanks.

**Comment by oliverw** [comment]
Not currently exposed by API.

**Reply by PMPMining** [reply]
Would like to add this functionality, which CS files should i be looking at ?

**Comment by coolman888** [comment]
Can you add this in new release? Thank you very much.
